ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -520,7 +520,9 @@ objects:
         function teardown() {
           set +e
           export PATH=$PATH:/tmp/shared
+          source /tmp/shared/ovirt-lease.conf
           source /etc/openshift-installer/ovirt.conf
+          export OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE=""
```

Comment:
why do we need an empty value?

Reply:
Look at the condition on [1]:
`RemoveTemplate: ok && customImage != ""`

I want to be false on CI with the way it is now and after the fix which will be:
`RemoveTemplate: !ok`

If it will be a full value then PRs will remove the CI templates until the fix will be merged to all releases.

[1] https://github.com/openshift/installer/blob/master/pkg/asset/cluster/ovirt/ovirt.go#L18